pkgs/by-name/me/memray/package.nix

```diff
@@ -48,7 +48,7 @@ python3Packages.buildPythonApplication rec {
       pytest-textual-snapshot
       pytestCheckHook
     ]
-    ++ lib.optionals (pythonOlder "3.12") [ greenlet ];
+    ++ lib.optionals (pythonOlder "3.14") [ greenlet ];
```

Comment:
Should be 3.14, otherwise test_greenlet.py fails. 

https://github.com/bloomberg/memray/blob/a9237b4b3fc5f83df40bd914a18c7f7a18743270/setup.py#L116